### ISSUE TITLE
Promote DLP Leaflet on Facebook

### ISSUE THREAD

**Issue by emcglackenr1** [CodeYourFuture/CYF-Staff-Tickets#907]
**Background**
Some of the organisations found for Sheffield outreach are Facebook only. Other organisations probably use Facebook too. So, this is a good opportunity to share the DLP leaflet and get sign-ups or referrals.

Dependency #838 

- [x] Find relevant organisations on Facebook - @emcglackenr1 [see sheet](https://docs.google.com/spreadsheets/d/1ivm9ioUt9ps4KYbPNnP4hX7uUvlo6juenWLmJMruoR8/edit#gid=0)
- [x] Get access to the CYF page so we can post from there (according to Alec a facebook account is required and this is a blocker. See updates below)
- [x] Contact them using the template from Amy's handover doc

We can ask permission to post the flyer. 
Relates to (stages described in Amy's [handover doc ](https://docs.google.com/document/d/1uCFk-eF7YEOns7kin4ssPz53MgMLrzSiw0Uqcvm0-FA/edit). )

**Important resources:**

- Organisations are on this [Spreadsheet](https://docs.google.com/spreadsheets/d/1ivm9ioUt9ps4KYbPNnP4hX7uUvlo6juenWLmJMruoR8/edit#gid=0) on the tab: ‘Partner organisations’ Sheffield - Trainee outreach .
- Progress on contacting them should be added to the spreadsheet as well. Any organisations contacted via another ticket should be updated to reduce duplication of work.
- Email template for contacting [[TEMPLATE] Sheffield Outreach](https://docs.google.com/document/d/1FBbmFvIdCZQO7bvB3FNypuwUZ1VqxKZ88PsEGQ0Wj1c/edit) this can be amended slightly to be sent as a Facebook message. 

**Further Background**
Organisation based in Sheffield that could potentially have volunteers or trainees for DLP need to be contacted.
_'We will need to start creating awareness about the program in the area among the communities we will be targeting. We need to reach out to organisations in the area that work with similar demographics to us.'_

Work on this is visible on other tickets on the [Launching Sheffield Chapter Milestone.](https://github.com/CodeYourFuture/CYF-Staff-Tickets/milestone/14)

This issue is based on the tasks from Amy's [handover doc ](https://docs.google.com/document/d/1uCFk-eF7YEOns7kin4ssPz53MgMLrzSiw0Uqcvm0-FA/edit).

**Comment by emcglackenr1** [comment]
New blocker: the account I set up has been suspended. 
Related Slack post: https://codeyourfuture.slack.com/archives/C029341FPH6/p1712678199109149

All I did was: 
- Set it up
- Put a CYF picture as my profile picture
- Accept Alec's invite to be a group administrator on the CYF page

My guess is the picture I used was flagged as 'not human' (see below) and this was done automatically. But I've asked in the PROM channel if someone else who has a Facebook can do this instead, I've messaged @alecmccrindle and I've appealed the suspension.

Matt let me know he has an active Facebook account and could take this over so I have reassigned. 

![Image](https://github.com/CodeYourFuture/CYF-Staff-Tickets/assets/64873698/85cd7e19-ec70-4b7f-a31f-8637de20e9c6)
